### PR TITLE
fix: fire transition events for reduced motion

### PR DESCRIPTION
**Pull request by geoffrich**
# Alaska Airlines Pull Request

Prerequisite to fix https://github.com/AlaskaAirlines/auro-accordion/issues/42.

## Summary:

The reduced motion settings disabled transitions, which caused events like `transitionend` not to play. This affected auro-accordion logic (see related issue for more details).

Instead, I set transition-duration to a small value so that transitions still play, but appear instantaneous. 

## Type of change:

Please delete options that are not relevant.

- [x] Revision of an existing capability

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Comment by geoffrich** [comment]
Once merged, I'll open an auro-accordion PR that updates this dep. This should resolve the issue.

File: src/_core.scss [hunk 1]
```diff
@@ -35,8 +35,9 @@ $focus-visible: null !default;
     box-sizing: border-box;
 
     @media (prefers-reduced-motion: reduce) {
-      transition: none !important;
-      animation: none !important;
+      animation-duration: 0.01ms !important;
```
Comment (blackfalcon) [inline src/_core.scss:38]: This needs to be better documented in the code that this IS why this was coded this way. This PR is only describing the WHAT and it's not clear as to the WHY. 

The comments in this code feed the automated docs. Please be sure to run `build:sassdoc` in order to update the automated docs.

**Review by blackfalcon — CHANGES_REQUESTED** [review]
Do you know why there is a 20k diff with package-lock?

**Comment by geoffrich** [comment]
Ah, sorry @blackfalcon I committed that accidentally. Will revert and force push a fresh commit.

**Review by blackfalcon — DISMISSED** [review]

**Review by blackfalcon — APPROVED** [review]

**Pull request merged**